app.py

```diff
@@ -57,7 +58,7 @@ def create_article():
             db.session.add(article)
             db.session.commit()
             return redirect('/posts')
-        except:
+        finally:
```

Comment:
Please, dont use finally
finally case execute every time in error or successful flows

Read this one, please https://www.programiz.com/python-programming/exception-handling